src/main/java/gregtech/common/command/CommandRecipeCheck.java

```diff
@@ -103,7 +105,7 @@ public void execute(@Nonnull MinecraftServer server, @Nonnull ICommandSender sen
             GTLog.logger.info("[Recipe Checker] Found " + count + " potential conflicts");
             for (Map.Entry<RecipeMap<?>, Object2ObjectOpenHashMap<Recipe, Set<Recipe>>> recipeMap : mismatchedRecipes.entrySet()) {
                 GTLog.logger.error(
-                        "\nIn map [" + recipeMap.getKey().unlocalizedName + "]");
+                        "\n[In Recipe map] :\"" + recipeMap.getKey().unlocalizedName + "\"");
```

Comment:
This commit is a big improvement for code readability.

Just wanted to mention though that log4j 2 supports formatted syntax where you don't have to manually inject variables into the message string. Check out https://logging.apache.org/log4j/2.x/manual/api.html, subheading "Substituting Parameters" for a bit of an overview. You can also use lambdas as parameters, which is useful for lazy evaluation of parameter expressions.